.github/workflows/e2e.yml

```diff
@@ -0,0 +1,42 @@
+name: e2e
+
+on:
+  workflow_dispatch: ~
+  schedule:
+    - cron: "0 4,16 * * 1-5"
+  push:
+    branches:
+      - main
+
+jobs:
+  integration-test:
```

Comment:
NIT

```suggestion
  e2e-test:
```

Reply:
The thing is.. in the other workflow in #728 there is a job called e2e-test aswell and I wanted to make it distinct.

In this case the full status check will ne named `e2e / Integration-test`

Reply:
https://github.com/elastic/synthetics/pull/730/commits/3d17d9665e656983059bdaaa8f3f9793e337bdf2